server.ts

```diff
@@ -3,16 +3,20 @@
  * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
  * In particular, applicationinsights automatically collects bunyan logs
  */
-import { initialiseAppInsights, buildAppInsightsClient } from './server/utils/azureAppInsights'
+import { initialiseAppInsights } from './server/utils/azureAppInsights'
+import createApplicationInfo from './server/applicationInfo'
 
-initialiseAppInsights()
-buildAppInsightsClient()
+const applicationInfo = createApplicationInfo()
```

Comment:
creating this here and then using dependency injection to pass through to where needed